modules/core/src/main/scala/com.snowplowanalytics.snowplow.bigquery/processing/Processing.scala

```diff
@@ -151,6 +152,7 @@ object Processing {
         _ <- Logger[F].debug(s"Processing batch of size ${events.size}")
         v2NonAtomicFields <- NonAtomicFields.resolveTypes[F](env.resolver, entities, env.schemasToSkip ::: env.legacyColumns)
         legacyFields <- LegacyColumns.resolveTypes[F](env.resolver, entities, env.legacyColumns)
```

Comment:
Nit: if `NonAtomicFields.resolveTypes` already has some Iglu failures we don't need to do this step